src/txn/txn_timestamp.c

```diff
@@ -403,6 +403,18 @@ __wt_txn_global_set_timestamp(WT_SESSION_IMPL *session, const char *cfg[])
         return (0);
 
 set:
+    /*
+     * Even if the timestamps have been forcibly set, they must always satisfy the condition that
+     * oldest <= stable. Don't fail as MongoDB violates this rule is very specific scenarios.
```

Comment:
typo: in very specific scenarios.

Reply:
Thanks! Fixed in cd66f55cc0151af8440e99f0186e6159f17a371e.